src/libsystemd/sd-journal/journal-verify.c

```diff
@@ -1307,15 +1336,6 @@ int journal_file_verify(
                   (unsigned long long) f->last_stat.st_size,
                   100 * p / f->last_stat.st_size);
 
-        if (data_fd >= 0)
```

Comment:
`safe_close()` on success path in the above can be also dropped.

Reply:
Duh, nice catch.